### DISCUSSION TITLE
TIFF format support in plot export

### DISCUSSION THREAD

**Discussion by abdullahahmed-dev** [posit-dev/positron#4689] [Feedback]
I recently switched from RStudio to Positron and so far loving the experience. One thing I miss is the ability to export plot to .TIFF format. I usually make graphs for my academic papers and most medicine journals require me to upload images in TIFF format. Please bring the ability to export plots/graphs/images in .tiff format.

**Comment by petetronic** [comment]
Thankyou @quantumfrost00 for trying out Positron! I created an [issue](https://github.com/posit-dev/positron/issues/4690) to track this enhancement request and marked it for triage.

**Reply by abdullahahmed-dev** [reply]
@petetronic Thankyou for acknowledging.
